identity/balance.go

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package identity
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+// BalanceRegistry provides amount of money that identity have on the balance in the blockchain
+type BalanceRegistry func(Identity) (uint64, error)
```

Comment:
Since it is just 1 function, I would name it with verb, like BalanceCheck, it is always better readable to see some action if you getting some data.

Reply:
Changed it to `Balance` since "Effective go" [recomends](https://golang.org/doc/effective_go.html#Getters) such approach for getters.